### PR TITLE
(test) added tests for get and post route

### LINKED ISSUE #6: Setting up Tests
Tests are an integral part of building framework. I would like to take up this task of setting up basic tests.

### PR DESCRIPTION
**Pull request by aniketbiprojit**
Closes #6 for now.

### CODE REVIEW

**Comment by aniketbiprojit** [comment]
Let me know if more tests are needed right now @disizali

File: tests/index.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { Application } from '../application.ts'
+import { assertEquals } from "https://deno.land/std/testing/asserts.ts";
```
Comment (disizali) [inline tests/index.ts]: for stability, can you use tagged releases URL, please?
```suggestion
import { assertEquals } from "https://deno.land/std@0.57.0/testing/asserts.ts";
```

File: tests/index.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+const port = 8080
+const app = new Application({ port: 8080 })
+
+app.start()
+
+async function request(suffix: string = '') {
```
Comment (disizali) [inline tests/index.ts]: this function needs to be flexible in methods, you can pass a parameter for it like "method";
```suggestion
async function request(suffix: string = '', method: "GET" | "POST" ) {
```

File: tests/index.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+		return 'Hello, World!'
+	})
+	const content = await request()
+
+    assertEquals(content,'Hello, World!')
+})
+
+Deno.test('Post Route', async () => {
+	app.post('/', (req: any) => {
+        assertEquals(req.method,'GET')
```
Comment (disizali) [inline tests/index.ts]:
```suggestion
        assertEquals(req.method,'POST')
```

File: tests/index.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+		return 'Hello, World!'
+	})
+	const content = await request()
```
Comment (disizali) [inline tests/index.ts]: actually this test doesn't visit even, you should call POST method on the request function.
```suggestion
	const content = await request("","POST");
```

**Review by disizali — COMMENTED** [review]
thank you @aniketbiprojit , I'm glad to see your cooperation.
just a few comments ...

**Comment by aniketbiprojit** [comment]
I will make another request.

**Pull request closed without merging**